core/src/test/java/com/linecorp/armeria/internal/annotation/AnnotatedHttpServiceFactoryTest.java

```diff
@@ -122,6 +131,35 @@ public void ofUserDefinedRepeatableDecorator() throws NoSuchMethodException {
         assertThat(udd2.value()).isEqualTo(2);
     }
 
+    @org.junit.jupiter.api.Test
+    public void testFindAnnotatedServiceElementsWithPathPrefixAnnotation() {
+        final Object object = new PathPrefixServiceObject();
+        final List<AnnotatedHttpServiceElement> elements = find("/", object, new ArrayList<>());
+
+        final List<String> paths = elements.stream()
+                                           .map(AnnotatedHttpServiceElement::route)
+                                           .map(Route::paths)
+                                           .flatMap(Collection::stream)
+                                           .collect(Collectors.toList());
+
+        assertThat(paths).allMatch(path -> path.startsWith(HOME_PATH_PREFIX + '/'));
+    }
+
+    @org.junit.jupiter.api.Test
+    public void testFindAnnotatedServiceElementsWithoutPathPrefixAnnotation() {
+        final Object serviceObject = new ServiceObject();
+        final String root = "/home";
```

Comment:
Could be replaced with `HOME_PATH_PREFIX`?

Reply:
Sure.